src/UsgsAstroLsSensorModel.cpp

```diff
@@ -2664,57 +2660,56 @@ std::string UsgsAstroLsSensorModel::constructStateFromIsd(const std::string imag
 
    int num_params = NUM_PARAMETERS;
 
-   state["m_imageIdentifier"] = isd.at("IMAGE_ID");
-   state["m_sensorType"] = isd.at("SENSOR_TYPE");
-   state["m_totalLines"] = isd.at("TOTAL_LINES");
-   state["m_totalSamples"] = isd.at("TOTAL_SAMPLES");
+   state["m_imageIdentifier"] = "UNKNOWN";
+   state["m_sensorType"] = "LINE_SCAN";
+   state["m_totalLines"] = isd.at("image_lines");
+   state["m_totalSamples"] = isd.at("image_samples");
    state["m_offsetLines"] = 0.0;
    state["m_offsetSamples"] = 0.0;
-   state["m_platformFlag"] = isd.at("PLATFORM");
-   state["m_aberrFlag"] = isd.at("ABERR");
-   state["m_atmRefFlag"] = isd.at("ATMREF");
+   state["m_platformFlag"] = 1;
+   state["m_aberrFlag"] = 0;
+   state["m_atmRefFlag"] = 0;

```

Comment:
I had a note about whether we should turn off or support. So :+1: on turning these off.

Reply:
If these values don't matter, do we still need to set them at all?

Reply:
The aberration correction and atmosphere flags can be removed. The platform identifier is actually part of the CSM spec.

You're right the next step after this PR is to prune out a lot of these un-needed values.